searx/engines/xpath.py

```diff
@@ -208,11 +209,14 @@
 
 '''
 
+categories = []
+'''engine dependent config'''
+
 
 def request(query, params):
     '''Build request parameters (see :ref:`engine request`).'''
     lang = lang_all
-    if params['language'] != 'all':
+    if params.get('language', 'all') != 'all':
```

Comment:
These param keys are set with defaults .. no need to safety *get*.

https://github.com/searxng/searxng/blob/f6f622f7e5627cc5afc8919a8fbc451b6003c9d7/searx/engines/__init__.py#L29C1-L48